notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>What is represented here?  </p><p>Also, same problem with plots as before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>The same but for ON data, substitued by a hist2d of reco_src_x, reco_src_y</p>

Reply:
> The same but for ON data, substitued by a hist2d of reco_src_x, reco_src_y

Ok, but the notebook is not updated, right? I cannot see the new plot. This reco_srcx, y, are the true source position? If so, 'reco' is not a good label - but not this PR's fault.

Reply:
<p>It seems that because this part of the notebook changes significantly, reviewnb doesn't allow to go to the latest version of the notebook in this conversation. You can see the lastest version of the notebook clicking in "Changes" from the reviewNB screen. </p><p>Regarding the source position, the plots are hist2d of RECONSTRUCTED positions, not true source position. </p>

Reply:
> It seems that because this part of the notebook changes significantly, reviewnb doesn't allow to go to the latest version of the notebook in this conversation. You can see the lastest version of the notebook clicking in "Changes" from the reviewNB screen.
> 

Ok, I see it now, including the sensitivity plot.

> Regarding the source position, the plots are hist2d of RECONSTRUCTED positions, not true source position.

But if these are real data, why are they so much clustered around the camera center?

Reply:
Because those are selected events after cuts in gammaness and theta2, the clustering is effect of the cut in theta2 I assume,

Reply:
> Because those are selected events after cuts in gammaness and theta2, the clustering is effect of the cut in theta2 I assume,

Ah, ok, surely that is the reason, thx!